experiments/gromacs/experiment.py

```diff
@@ -42,6 +42,12 @@ class Gromacs(
     )
 
     def compute_applications_section(self):
+        # Default values
+        self.add_experiment_variable("n_ranks", 1, True)
+        target = "cpu"
+        bonded_target = "cpu"
+        npme = "0"
+
```

Comment:
Fixes error for `gromacs+single_node` https://github.com/LLNL/benchpark/actions/runs/14066858759/job/39391533278?pr=701

Reply:
Being addressed in #714